OpenRA.Mods.Common/Traits/Render/WithIdleOverlay.cs

```diff
@@ -67,7 +70,11 @@ public WithIdleOverlay(Actor self, WithIdleOverlayInfo info)
 
 			buildComplete = !self.HasTrait<Building>(); // always render instantly for units
 			overlay = new Animation(self.World, rs.GetImage(self));
-			overlay.PlayRepeating(RenderSprites.NormalizeSequence(overlay, self.GetDamageState(), info.Sequence));
+			if (Info.StartSequence != null)
+				overlay.PlayThen(Info.StartSequence,
```

Comment:
This sequence needs to be normalized to avoid breakage when actors are spawned with < 100% health.  Please also fix that in `WithSpriteBody` if that needs it, and you can earn some bonus points if you fix `Info` &rarr; `info` while you're in these parts of the code.